raiden/billing/invoices/handlers/invoice_handler.py

```diff
@@ -0,0 +1,23 @@
+from eth_utils import encode_hex
+
+from raiden.encoding.messages import DEFAULT_PAYMENT_INVOICE_HASH
+from raiden.billing.invoices.constants.invoice_status import InvoiceStatus
+
+
+def handle_received_invoice(storage, payment_hash_invoice):
```

Comment:
We should not add a validation of an invoice in a PAID status? that will mean the payee is trying to repay an invoice (probably by error) so we should throw an error

Reply:
ok ready